lib/support/live_select_web/live/showcase_live.ex

```diff
@@ -300,9 +355,16 @@ defmodule LiveSelectWeb.ShowcaseLive do
       {:ok, settings} ->
         socket.assigns
 
+        attrs =
+          if settings.mode == :quick_select do
+            %{selected_option_class: "cursor-pointer font-bold hover:bg-gray-400 rounded"}
+          else
+            %{}
+          end
+
         socket =
           socket
-          |> assign(:settings_form, Settings.changeset(settings, %{}) |> to_form)
+          |> assign(:settings_form, Settings.changeset(settings, attrs) |> to_form)
```

Comment:
Suggestion: let's remove this code and all other changes in this file related to the `selected_option_class` override, and add this:

```
params =
      if params["mode"] == "quick_tags" do
        Map.put_new(
          params,
          "selected_option_class",
          "cursor-pointer font-bold hover:bg-gray-400 rounded"
        )
      else
        params
      end
```

just before we create the changeset. Advantages:

1. simpler
2. more transparent, as the user will see the classes in the UI's styling options
3. the user can override these default classes if they want to

Reply:
Yeah, what I had done was kind of a hack to just to make it work. 
This is nice, but it seems to not reset when you change back to tags mode.
Maybe do a reset of `selected_option_class` in the else clause or just let it be?